WBooks/Views/BookIndex/BookIndexViewModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  BookIndexViewModel.swift
+//  WBooks
+//
+//  Created by Diego Quirós on 3/13/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+import ReactiveSwift
+
+internal final class BookIndexViewModel {
+
+    var bookViewModels: MutableProperty<[BookViewModel]> = MutableProperty([])
```

Comment:
Generally we avoid exposing mutable properties and only use them internal unless you really want somebody else to mutate this property. The general pattern looks like this

```swift
  private(set) var _bookViewModels: MutableProperty<[BookViewModel]> = MutableProperty([])
  internal var bookViewModels: Property<[BookViewModel]> {
    return Property(capturing: _bookViewModels)
  }
```

We are using `private(set)` to make the instance variable readonly from the external point of view, only code inside the class can change the value.